base_layer/core/src/transactions/transaction_components/transaction_output_version.rs

```diff
@@ -59,7 +58,6 @@ impl TryFrom<u8> for TransactionOutputVersion {
         match value {
             0 => Ok(TransactionOutputVersion::V0),
             1 => Ok(TransactionOutputVersion::V1),
-            2 => Ok(TransactionOutputVersion::V2),
             _ => Err("Unknown version!".to_string()),
         }
     }
```

Comment:
ditto single version

Reply:
See above

Reply:
Ah ok, used for testing.
- Maybe consistently apply comment `/// Currently only used in tests, this can be used as the next version`
- `pub enum TransactionKernelVersion` is missing its test `V1 = 1,`